admin_ui/cypress/integration/LoginAndLogout.spec.ts

```diff
@@ -42,6 +48,8 @@ context('Startup', () => {
         cy.location('pathname', { timeout: 2000 }).
             should('eq', '/');
 
+        cy.wait(3000);
```

Comment:
I think we can drop this, as it's at the end of the test shouldn't be required.

Reply:
I think we should leave this line because without this line I get an error in the test. Do you want to remove the delete tests or is restarting the server ok?

Reply:
OK, fair enough - let's keep it for now then.